Tests/SwiftProtobufTests/Test_FuzzTests.swift

```diff
@@ -0,0 +1,124 @@
+// Tests/SwiftProtobufTests/Test_FuzzTests.swift - Test cases from fuzz testing
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See LICENSE.txt for license information:
+// https://github.com/apple/swift-protobuf/blob/master/LICENSE.txt
+
+// These are input cases from fuzz testing that caused crashes/failures,
+// captured as unittest to debug, watch for regressions, etc. Generally how it
+// decodes doesn't matter, just that it can/can't decode without
+//
+// If a failure is found via manually running the binaries, the failures end
+// with a summmary that shows the data output is hex pairs, an escaped string,
+// written to a file, and then also as Base64. You can directly get the hex
+// values from there, or can use xxd to dump from the file.
+//
+// For failures that get filed by oss-fuzz, the bugs will have links to detailed
+// reports, those have a test case (and reduced test case) that can be download.
+//
+// Easily get the hex dumps via `xxd -i < _test_case_file_`
+
+import Foundation
+import XCTest
+
+class Test_FuzzTests: XCTestCase {
+
+  func assertBinaryFails(_ bytes: [UInt8], file: XCTestFileArgType = #file, line: UInt = #line) {
+    do {
```

Comment:
Can these failing helpers be condensed down to the one-liner:

```swift
XCTAssertThrowsError(try Fuzz_Testing_Message(...), file: file, line: line)
```

Reply:
done